main/app/logger/log.cpp

```diff
@@ -41,7 +41,20 @@ CLog& CLog::getInstance() {
 void CLog::mInit() {}
 
 void CLog::mWriteLog(const char* apScope, const std::string& arText, LogType aType) {
-    auto timeinfo = CTime::mGetTime();
+    struct tm timeinfo;
+    try {
+        timeinfo = CTime::mGetTime();
+    }
+    catch (const std::runtime_error& e) {
```

Comment:
fix tijd. ipv 1970 -> time not initialized.

Reply:
Implemented
![image](https://user-images.githubusercontent.com/98885494/165317629-6037727f-94c3-4406-aab4-4761b70e8f03.png)